packages/restitution/src/Restitution.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+  withClassDefault,
+  withClassModifier,
+  WithClassModifierOptions,
+} from '@axa-fr/react-toolkit-core';
+import { compose } from 'recompose';
+
+const DEFAULT_CLASSNAME = 'af-restitution__listdef';
+
+type RestitutionBaseProps = {
+  label: string;
+  value?: string;
+  values?: string[];
+  className?: string;
+}
+
+type RestitutionValuesProps = {
+  values: string[];
```

Comment:
Use _RestitutionValuesProps_ into _RestitutionBaseProps_  to make composition & clarify mandatory props
```javascript
type RestitutionBaseProps = {label: string} & Partial<{value: string; className: string> & Partial<RestitutionValuesProps>
```